config.yaml

```diff
@@ -72,12 +76,11 @@ collate:
   mem: 16GB
   ncpus: 4
   queue: normal
-  exe: /g/data/vk83/apps/spack/0.20/release/linux-rocky8-x86_64/intel-19.0.5.281/mom5-git.2023.11.09=2023.11.09-ewcdbrfukblyjxpkhd3mfkj4yxfolal4/bin/mppnccombine.spack  
-
+  exe: /g/data/vk83/apps/spack/0.20/release/linux-rocky8-x86_64/intel-19.0.5.281/mom5-git.2023.11.09=2023.11.09-64l5azdtcoxhrgb5ynn2vued5lmjvn33/bin/mppnccombine.spack
```

Comment:
This is still failing on the check because it isn't referencing `mppnccombine-fast`

Reply:
referencing back to `mppnccombine-fast`.